include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,563 @@
+    // [Future TODO]:
+    //VkBool32           variablePointersStorageBuffer;
+    //VkBool32           variablePointers;
+    
+    //VkBool32           protectedMemory;
+    //VkBool32           samplerYcbcrConversion;
+    bool shaderDrawParameters = false;
+
+
+
+
+    /* Vulkan 1.2 Core */
+
+    bool samplerMirrorClampToEdge = false;          // ALIAS: VK_KHR_sampler_mirror_clamp_to_edge
+    bool drawIndirectCount = false;                 // ALIAS: VK_KHR_draw_indirect_count
+
+    // or VK_KHR_8bit_storage:
+    //VkBool32           storageBuffer8BitAccess;
+    //VkBool32           uniformAndStorageBuffer8BitAccess;
+    //VkBool32           storagePushConstant8;
+    
+    // or VK_KHR_shader_atomic_int64:
+    //VkBool32           shaderBufferInt64Atomics;
+    //VkBool32           shaderSharedInt64Atomics;
+   
+    // or VK_KHR_shader_float16_int8:
+    //VkBool32           shaderFloat16;
+    //VkBool32           shaderInt8;
+    
+    // or VK_EXT_descriptor_indexing
+    //VkBool32           descriptorIndexing;
+    //VkBool32           shaderInputAttachmentArrayDynamicIndexing;
+    //VkBool32           shaderUniformTexelBufferArrayDynamicIndexing;
+    //VkBool32           shaderStorageTexelBufferArrayDynamicIndexing;
+    //VkBool32           shaderUniformBufferArrayNonUniformIndexing;
+    //VkBool32           shaderSampledImageArrayNonUniformIndexing;
+    //VkBool32           shaderStorageBufferArrayNonUniformIndexing;
+    //VkBool32           shaderStorageImageArrayNonUniformIndexing;
+    //VkBool32           shaderInputAttachmentArrayNonUniformIndexing;
+    //VkBool32           shaderUniformTexelBufferArrayNonUniformIndexing;
+    //VkBool32           shaderStorageTexelBufferArrayNonUniformIndexing;
+    //VkBool32           descriptorBindingUniformBufferUpdateAfterBind;
+    //VkBool32           descriptorBindingSampledImageUpdateAfterBind;
+    //VkBool32           descriptorBindingStorageImageUpdateAfterBind;
+    //VkBool32           descriptorBindingStorageBufferUpdateAfterBind;
+    //VkBool32           descriptorBindingUniformTexelBufferUpdateAfterBind;
+    //VkBool32           descriptorBindingStorageTexelBufferUpdateAfterBind;
+    //VkBool32           descriptorBindingUpdateUnusedWhilePending;
+    //VkBool32           descriptorBindingPartiallyBound;
+    //VkBool32           descriptorBindingVariableDescriptorCount;
+    //VkBool32           runtimeDescriptorArray; // [FUTURE TODO]
+    
+    bool                 samplerFilterMinmax = false;   // ALIAS: VK_EXT_sampler_filter_minmax
+    
+    //VkBool32           scalarBlockLayout;     // or VK_EXT_scalar_block_layout // [FUTURE TODO]
+    
+    //VkBool32           imagelessFramebuffer;  // or VK_KHR_imageless_framebuffer 
+    
+    //VkBool32           uniformBufferStandardLayout;   // or VK_KHR_uniform_buffer_standard_layout
+    
+    // Don't expose:
+    //VkBool32           shaderSubgroupExtendedTypes;   // or VK_KHR_shader_subgroup_extended_types
```

Comment:
`AMD_shader_ballot`

Reply:
why did I initially tagged this as DO NOT EXPOSE ?! I can't recall